src/pages/DashBoard/DashBoard.js

```diff
@@ -1,8 +1,116 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import ItemCard from '../../components/ItemCard/ItemCard';
+import AdCard from './AdCard/AdCard';
 import './DashBoard.scss';
+import UseALink from './UseALink/UseALink';
 
 const DashBoard = props => {
-  return <>DashBoard</>;
+  const [items, setItems] = useState([]);
+  const [recommendItems, setRecommendItems] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/perfumes.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        setItems(data);
+      });
```

Comment:
```suggestion
    fetch('/data/perfumes.json')
      .then(res => res.json())
      .then(data => {
        setItems(data);
      });
```
fetch 함수에서 GET 메서드는 default 값 이므로, 삭제할 수 있습니다!